query-coordinator/docker/query-coordinator.conf.j2

```diff
@@ -5,6 +5,28 @@ com.socrata.query-coordinator {
   liveness-check.port = 6031
 
   all-secondary-instance-names = [{{ SECONDARY_INSTANCES }}]
+  mirrors = {
+    {% if MIRROR_PRIMARY_1 is defined -%}
+      "{{MIRROR_PRIMARY_1}}" = {{MIRROR_1}}
```

Comment:
I know you said this already works, but I find it weird that you're checking if something is defined and then assigning to it, instead of assigning something else its value. Should it not be if MIRROR_1 is defined? or maybe MIRROR_1 = MIRROR_PRIMARY_1?

Reply:
resolved in slack. for posterity -- that's template expansion.